docker-compose.yml

```diff
@@ -26,6 +26,12 @@ services:
     image: "quay.io/galexrt/zulip:1.8.1-0"
     build:
       context: .
+      args:
+        # Use these if you want to download zulip from a different repo/branch
+        # ZULIP_GIT_URL:
+        # ZULIP_GIT_REF:
```

Comment:
What do you think about including the default values here, to help make it feel easier to edit?